app/views/bonus_engine/api/points/update.rabl

```diff
@@ -0,0 +1,3 @@
+object @point
```

Comment:
you could move all these attributes to another partial ... mm node or something, then just call for it in the create.rabl and update.rabl as both are identical:

```
object @point
extends "bonus_engine/api/points/point"
```